README.md

```diff
@@ -5,12 +5,45 @@ Fixity is a simple commandline tool that assists in checking fixity for AIPs sto
 
 The Archivematica Storage Service provides a REST API endpoint for checking fixity of AIPs. This tool provides a simple commandline interface to that functionality, in a way that's simple to automate.
 
+
+
+Installation
+------------
+
+- Checkout or link the code to /usr/lib/archivematica/fixity
+- Create a virtualenv in /usr/share/python/fixity, and install fixity and dependencies in it
+
+  ```
+  # virtualenv /usr/share/python/fixity
+  # source /usr/share/python/fixity/bin/activate
```

Comment:
I think this need something to make clear this is running as root and not just a comment. Maybe prepend root@host? Or change `#` to `$`? Or remove the command prompts & virtualenv stuff?